config/locales/en.yml

```diff
@@ -94,19 +94,28 @@ en:
         pagination_description: '%{current_page} of %{total_pages}'
       insight_summary:
         data_types:
-          - Qualitative
-          - Quantitative
+          - keyword: Qualitative
```

Comment:
These changes have actually caused a couple of the Rspec tests to fail. @margOnline might be the best person to align with on this as I believe she has the best knowledge of the presenter and it's tests